prow/cluster/jobs/IBM/go-repo-template/IBM.go-repo-template.master.yaml

```diff
@@ -116,20 +116,20 @@ postsubmits:
         name: ""
         securityContext:
           privileged: true
-  - name: images_go-repo-template_postsubmit
-    branches:
-    - ^master$
-    decorate: true
-    path_alias: github.com/IBM/go-repo-template
-    labels:
-      preset-service-account: "true"
-    spec:
-      containers:
-      - command:
-        - entrypoint
-        - make
-        - images
-        image: quay.io/multicloudlab/multicloudlab-builder:v20190927-1ed9e5c
-        name: ""
-        securityContext:
-          privileged: true
+  # - name: images_go-repo-template_postsubmit
```

Comment:
can you delete this if not needed?

also how does this PR enabled travis to build image?

Reply:
This PR ensure the PR in `go-repo-template` will not be merged until travis ci is passed.

Reply:
done